core/templates/components/state-directives/answer-group-editor/answer-group-editor.directive.html

```diff
@@ -54,16 +54,19 @@
     width: 100%;
     margin-top: 4%;
   }
+  answer-group-editor .oppia-answer-rule-block {
+    border-bottom: 1px solid #ccc;
+  }
 </style>
 
-<div style="height: 100%;">
+<div class="h-100">
   <!-- For the default answer group, 'rules' is null. Otherwise, 'rules' is a list of rules. -->
   <div ng-if="$ctrl.rules">
     <strong>
       If the learner's answer...
     </strong>
     <ul class="nav oppia-option-list nav-stacked nav-pills" role="tablist" ng-model="$ctrl.rules">
-      <li ng-repeat="rule in $ctrl.rules" ng-class="{'active': $ctrl.activeRuleIndex === $index}" class="oppia-rule-block oppia-sortable-rule-block protractor-test-rule-block" style="margin-top: 0;" ng-style="$last ? {'border-bottom': '1px solid #ccc'} : {}">
+      <li ng-repeat="rule in $ctrl.rules" ng-class="{'active': $ctrl.activeRuleIndex === $index} $last ? oppia-answer-rule-block : {}" class="mt-0 oppia-rule-block oppia-sortable-rule-block protractor-test-rule-block">
```

Comment:
I don't think this is how ng-class works. Please read up on it and handle it properly.